### PR TITLE
BROKERS: `Score` Update

### LINKED ISSUE #248: BROKERS: Update Score

### PR DESCRIPTION
**Pull request by Ulugbek4499**
closes #248

### CODE REVIEW

File: Tarteeb.Api/Brokers/Storages/StorageBroker.Scores.cs [hunk 1]
```diff
@@ -13,6 +13,9 @@ public partial class StorageBroker
     {
         DbSet<Score> Scores { get; set; }
 
+        public async ValueTask<Score> UpdateSocreAsync(Score score)=>
```
Comment (khabibulladev) [inline Tarteeb.Api/Brokers/Storages/StorageBroker.Scores.cs:16]: UpdateSocreAsync -> UpdateScoreAsync

**Review by khabibulladev — CHANGES_REQUESTED** [review]

File: Tarteeb.Api/Brokers/Storages/StorageBroker.Scores.cs (hunk 1 above)
Comment (Ulugbek4499) [reply Tarteeb.Api/Brokers/Storages/StorageBroker.Scores.cs]: Resolved

**Comment by GulchexrasGithub** [comment]
Resolve conflicts

**Review by jasurbekyusuf — COMMENTED** [review]
LGTM

**Pull request merged**